### ISSUE TITLE
ESLint couldn't find the plugin "eslint-plugin-react".

### ISSUE THREAD

**Issue by lornally** [eslint/eslint#13813]
**Tell us about your environment**

- Node version: v15.0.1
- npm version: v7.0.3
- Local ESLint version: Not found
- Global ESLint version: v7.12.1 (Currently used)

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
	"extends": [
    "eslint:recommended",
    "plugin:react/recommended"
],
	parserOptions: {
		sourceType: "module",
		allowImportExportEverywhere: true
	}
};

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```zsh
eslint b/*.jsx      

```

**What actually happened? Please include the actual, raw output from ESLint.**
```zsh
Oops! Something went wrong! :(

ESLint: 7.12.1

ESLint couldn't find the plugin "eslint-plugin-react".

(The package "eslint-plugin-react" was not found when loaded as a Node module from the directory "/Users/bergman/git/_X/code/基础项目/打包/pack实践/blank".)

It's likely that the plugin isn't installed correctly. Try reinstalling by running the following:

    npm install eslint-plugin-react@latest --save-dev

The plugin "eslint-plugin-react" was referenced from the config file in ".eslintrc.js".

If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.

```

and this two script can not fix it. 
```zsh
 cnpm install -g eslint-plugin-react 
 yarn add eslint-plugin-react@latest --dev
```

at first, i have a lot of project use same config file, at that time, the config file is in father's dir,  i can not fix it. 
but when i move the .eslintrc.js to every project dir, the bug is fixed. 
so eslint depend on the config file's dir ?

**Comment by aladdin-add** [comment]
you can use the [resolve-plugins-relative-to](https://eslint.org/docs/user-guide/command-line-interface#-resolve-plugins-relative-to)
